tsds-grafana-handler/tsds_grafana_handler.py

```diff
@@ -37,17 +37,55 @@ def get_timeframe(seconds):
 
         return "{0}s".format(seconds)
 
-def findtarget_names(tsds_result, alias_list, target_aliases):
+def is_metric(key):
+        ''' Returns true if key is a metric
         '''
+
+        if 'values.' in key:
+                return False
+        else:
+                return True
+
+def metric_label(tsds_result):
+        ''' Builds a graph label based on a tsds result
+
+        tsds_result tsds result containing datapoints and metric names
+        '''
+        # Get all metric names to populate as the target name.
+        targetname = []
+        nonValues  = []
+        for k, v in tsds_result.iteritems():
+                if is_metric(k):
+                        nonValues.append(k)
+        nonValues.sort(key=natural_keys)
+
+        # May need to handle utf8 conversions here?
+        for nV in nonValues:
+                if nV in tsds_result: targetname.append(tsds_result[nV])
```

Comment:
Why are we checking for nV in tsds_result? Based on the above, seems like this cannot be false. I might be missing something?